src/containers/App.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from "react";
+import { BrowserRouter, Route, Switch } from "react-router-dom";
+
+import Menu from "./Menu/Menu";
+import Upcoming from "./Upcomings/Upcomings";
+import Live from "./Live/Live";
+import Favorite from "./Fovorite/Favorite";
+import Leagues from "./Leagues/Leagues";
+import Standings from "./Standings/Standings";
+import { AppRoutes } from "../utils/consts";
+import Match from "./Match/Match";
+
+class App extends Component {
+  render() {
+    return (
+      <div>
+        <BrowserRouter>
+          <div>
+            <Menu />
+            <Switch>
+              <Route path={AppRoutes.upcomings} component={Upcoming} />
+              <Route path={AppRoutes.live} component={Live} />
+              <Route path={AppRoutes.favorite} component={Favorite} />
+              <Route exact path={AppRoutes.leagues} component={Leagues} />
+              <Route path={`${AppRoutes.leagues}:id/`} component={Standings} />
```

Comment:
я бы убрал финальный слэш из роутов, тогда здесь роуты будут выглядеть более натурально
```suggestion
              <Route path={`${AppRoutes.leagues}/:id`} component={Standings} />
```